dashboard/src/app/workspaces/workspace-details/environments/environments.controller.ts

```diff
@@ -111,6 +135,15 @@ export class WorkspaceEnvironmentsController {
     }
   }
 
+  /**
+   * Returns true if the recipe type is scalable.
+   *
+   * @returns {boolean}
+   */
+  isScalable(): boolean {
```

Comment:
Propose the term - isMultiContainer/isMultiMachine ?

Reply:
done